enclave-tls/src/crypto_wrappers/wolfcrypt/main.c

```diff
@@ -21,8 +21,13 @@ extern crypto_wrapper_err_t wolfcrypt_cleanup(crypto_wrapper_ctx_t *);
 
 static crypto_wrapper_opts_t wolfcrypt_opts = {
 	.api_version = CRYPTO_WRAPPER_API_VERSION_DEFAULT,
+#ifdef SGX
+	.name = "wolfcrypt_sgx",
```

Comment:
For SGX build, is it still necessary to emphasize "sgx"? Why not use the same name as normal build.